scripts/dnd5e-helpers.js

```diff
@@ -906,11 +946,38 @@ function ReactionDetect_preCreateChatMessage(msg){
     /** hand over to reaction apply logic (checks combat state, etc) */
     ReactionApply(speaker.actor, speaker.token, itemId);
   }
-}
+};
 
 Hooks.on("preCreateChatMessage", async (msg, options, userId) => {
  const reactMode = game.settings.get('dnd5e-helpers', 'cbtReactionEnable');
   if ( reactMode === 1 || reactMode === 3) {
     ReactionDetect_preCreateChatMessage(msg);
   }
+
+  let deathSaveRoll = msg.flavour.match(/Death Saving Throw/)
+  let rollTotal = parseInt(msg.content)
```

Comment:
does this need a null check? what does parseInt return if the content doesn't contain or contains multiple rolls?

Reply:
the double check of rollTotal and deathSaveRoll *should* prevent any oddness with double rolls...

I'll look over and check again

Reply:
updated checks for Death saving throw to read the roll flags data, and moved the content check to within that if() 
```js
 let rollType = msg.flags.dnd5e.roll
  if (rollType === "death" && (game.settings.get('dnd5e-helpers', 'owDeathSave'))) {
    if (parseInt(msg.content) < 6)
      let actor = game.actors.get(msg.speaker.actor)
    OpenWounds(actor)```

Reply:
awesome. that's what I was hoping for!